File: backend/src/main/java/com/example/Board/modal/responses/PagingResponse.java
package com.example.Board.modal.responses;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class PagingResponse {

    private List<BoardResponse> boards;
    private int totalPage;
}

File: frontend/src/models/responses/PagingResponse.ts
import BoardResponse from "./BoardResponse";

interface PagingResponse {
  boards: BoardResponse[];
  totalPage: number;
}

const emptyPagingResponse: PagingResponse = {
  boards: [],
  totalPage: 0,
};

export default PagingResponse;
export { emptyPagingResponse };
